### PR TITLE
Update Event Name input field

### PR DESCRIPTION
**Pull request by shaokiat**
Added Title to Event Name input field

InputField is basically the old InputRow but with title at the top left.

InputField without textarea:
image

InputField with textarea:
image

Figma reference:
image

https://trello.com/c/PBiRDzrk/5-update-input-text-area-field

### CODE REVIEW

File: frontend/src/components/Mobile/InputField.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  }
+  &.ant-input::placeholder {
+    color: #bfbfbf;
+    font-size: 0.8rem;
+    padding-left: 5px;
+  }
+`
+
+const { TextArea } = Input
+const StyledTextArea = styled(TextArea)<{ haserror?: boolean }>`
```
Comment (hui444) [inline frontend/src/components/Mobile/InputField.tsx:31]: consider using camelCase here too

File: frontend/src/components/Mobile/InputField.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  &.ant-input {
+    background: #f3f3f9;
+    width: 100%;
+    border-radius: 20px;
+    border: 1px solid ${(props) => (props.haserror ? 'red' : '#f3f3f9')};
+    padding: 5px 10px;
+    margin: 0px 0px 0px 0px;
```
Comment (hui444) [inline frontend/src/components/Mobile/InputField.tsx:38]: if the styling is very similar consider abstracting out using styled-components' css

File: frontend/src/components/Mobile/InputField.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    ${(props) => props.haserror && `background: #ffd1d1;`}
+  }
+  &.ant-input::placeholder {
+    color: #bfbfbf;
+    padding: 4px;
+    font-size: 0.8rem;
+  }
+`
+
+const StyledTitle = styled.text`
```
Comment (hui444) [inline frontend/src/components/Mobile/InputField.tsx:49]: i think styled.text is not a thing, consider using styled.p and removing the margin! the text element is only suppose to be used within the svg element

File: frontend/src/components/Mobile/InputField.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  font-family: Inter;
```
Comment (hui444) [inline frontend/src/components/Mobile/InputField.tsx:50]: revamp is using Lato!

File: frontend/src/components/Mobile/InputField.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  margin-right: 20px;
+  white-space: nowrap;
+`
+
+type InputRowProps = {
+  title?: string
+  placeholder: string
+  value?: string
+  setValue?: Dispatch<SetStateAction<string>> | ((input: string) => void)
+  textarea?: boolean
```
Comment (hui444) [inline frontend/src/components/Mobile/InputField.tsx:64]: consider using camelCase

File: frontend/src/components/Mobile/InputField.tsx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  haserror?: boolean
```
Comment (hui444) [inline frontend/src/components/Mobile/InputField.tsx:65]: this as well, consider using camelCase

File: frontend/src/routes/FacilityBooking/CreateBooking/index.tsx [hunk 7]
```diff
@@ -29,27 +29,16 @@ import {
 } from '../../../store/facilityBooking/action'
 import LoadingSpin from '../../../components/LoadingSpin'
 import { PATHS } from '../../Routes'
+import InputField from '../../../components/Mobile/InputField'
 
 const Background = styled.div`
-  background-color: #fafaf4;
+  background-color: #ffffff;
```
Comment (hui444) [inline frontend/src/routes/FacilityBooking/CreateBooking/index.tsx:35]: consider shortening this to just #fff

**Review by hui444 — CHANGES_REQUESTED** [review]
this should not merge into devel!

File: frontend/src/routes/FacilityBooking/CreateBooking/index.tsx (hunk 7 above)
Comment (shaokiat) [reply frontend/src/routes/FacilityBooking/CreateBooking/index.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx (hunk 6 above)
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx (hunk 5 above)
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx (hunk 4 above)
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx (hunk 3 above)
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx (hunk 1 above)
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+  width: 100%;
+  margin: 10px 0px;
+`
+
+const StyledInput = styled(Input)<{ haserror?: boolean }>`
+  &.ant-input {
+    background: #f3f3f9;
+    width: 100%;
+    border-radius: 30px;
+    border: 1px solid ${(props) => (props.haserror ? 'red' : '#f3f3f9')};
```
[the remark this answers is not in the document]
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

File: frontend/src/components/Mobile/InputField.tsx [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+    padding: 5px 10px;
+    margin: 0px 0px 0px 0px;
```
[the remark this answers is not in the document]
Comment (shaokiat) [reply frontend/src/components/Mobile/InputField.tsx]: 9abba80

**Pull request closed without merging**